config/services/wireguard.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>WireGuard</short>
+  <description>WireGuard is the simple, fast and modern VPN.</description>
```

Comment:
AFAIK, this service is only need on the VPN server side. Clients won't need it. If that's accurate then please add a sentence or two to the description stating as such.